Document-Processing/Excel/Spreadsheet/Javascript-ES6/notes.md

```diff
@@ -84,6 +146,8 @@ In the below example, the note functionality is disabled in the Spreadsheet.
 
 The notes can be added initially when the Spreadsheet loads using cell data binding. You need to use the `notes` property in the cell settings to add notes to the Spreadsheet.
 
+In the below example, you can navigate between notes using Previous Note and Next Note options, toggle individual note visibility with Show/Hide Note, and display all notes at once using Show All Notes.
```

Comment:
In the below example, you can navigate between notes using **Previous Note** and **Next Note** options, toggle individual note visibility with **Show/Hide Note** and display all notes at once using **Show All Notes**.